app/src/main/res/values-sk/strings.xml

```diff
@@ -11,5 +11,10 @@
     <string name="stats_filter_from">Od</string>
     <string name="stats_filter_to">Do</string>
     <string name="stats_filter_button">Filtrovať</string>
+    <string name="button_for_adding_a_new_spending">Pridaj nový výdavok</string>
+    <string name="back_button">Tlačidlo späť</string>
+    <string name="discard_changes_dialog_discard">Zahoď</string>
```

Comment:
Ak to je v dialógu tak by som možno navrhoval "Áno" , prípadne " Zahodiť "

Reply:
Otázka (v inom stringu) znie "Zahodiť zmeny?", preto by som nechal "Zahodiť", aby užívateľ videl rovnaké slovo a mohol rýchlejšie zareagovať.